### PR TITLE
event is deleted after cancel it

### PR DESCRIPTION
**Pull request by Andreslw**

### CODE REVIEW

File: src/partner/modules/EventList/EventListContainer.tsx [hunk 1]
```diff
@@ -4,8 +4,9 @@ import { EventList } from "./EventList";
 
 export interface IEventListContainerProps {
   events: IEvent[];
+  handleCancelEvent: Function;
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListContainer.tsx]: avoid using `Function`, if the function just takes any event a doesn't return anything use:

```ts
export interface IEventListContainerProps {
  handleCancelEvent: (e: any) => void;
}
```

else use a meaningful function signature

File: src/partner/modules/EventList/EventList.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { ListStyled } from "src/partner/modules/ui/";
 
 interface IEventListProps {
   events: IEvent[];
+  handleCancelEvent: Function;
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventList.tsx]: avoid using `Function`, if the function just takes any event a doesn't return anything use:

```ts
export interface IEventListContainerProps {
  handleCancelEvent: (e: any) => void;
}
```

else use a meaningful function signature

File: src/partner/modules/EventList/EventListItem.tsx [hunk 3]
```diff
@@ -2,15 +2,13 @@ import React from "react";
...
-import { Notification } from "src/partner/modules/ui/Modal/Notification";
 import { CreateEvent } from "src/components/event/Create";
-import styledComponents from "styled-components";
 import { ListStyled } from "src/partner/modules/ui";
-import { NotificationContext } from "src/providers";
 
 export interface IEventItemProps {
   eventInfo: IEvent;
   eventView?: boolean;
+  handleCancelEvent: Function;
```
Comment (mishelashala) [inline src/partner/modules/EventList/EventListItem.tsx]: avoid using `Function`, if the function just takes any event a doesn't return anything use:

```ts
export interface IEventListContainerProps {
  handleCancelEvent: (e: any) => void;
}
```

else use a meaningful function signature

**Review by mishelashala — CHANGES_REQUESTED** [review]

**Review by mishelashala — APPROVED** [review]

File: src/partner/views/EventView.tsx [hunk 4]
```diff
@@ -74,11 +74,14 @@ export class EventView extends React.Component<IEventViewProps, ICurrentEventsVi
...
-            eventView={true}
-          />
+          {
+            <EventListItem
+              handleCancelEvent={() => {}}
+              key={this.state.localEvent.id}
+              eventInfo={this.state.localEvent}
+              eventView={true}
+            />
+          }
```
Comment (mishelashala) [inline src/partner/views/EventView.tsx]: why the curly braces?

Comment (Andreslw) [reply src/partner/views/EventView.tsx]: component was commented

**Pull request merged**